### DISCUSSION TITLE
Theming R packages on the Epiverse-TRACE website?

### DISCUSSION THREAD

**Discussion by joshwlambert** [epiverse-trace/epiverse-trace.github.io#260] [General]
A point raised in today's Epiverse-TRACE development meeting was that it might be useful to have somewhere on the Epiverse-TRACE website (or somewhere else if more appropriate) that groups packages into themes. Some themes that users might find useful are: 

* package that provide inference methods
* packages that calculate summary metrics
* packages that export data
* packages that simulate data
* packages that clean and tag data

The packages we are developing will not fall into a single theme, or the theme might not completely describe the scope of a package. For example, {serofoi} can simulate data and provides inference methods.

Within each theme there could be information to guide users towards the package they require. An example could be, a user comes to Epiverse looking for a package that simulates transmission chain data. They click on the simulation theme, and the new web page provides a list of all Epiverse packages (and potentially other R packages in the epi community) that simulate data with a short description of what the package can do (with respect to the chosen theme) and maybe a link to the relevant pkgdown site for that package.

**Comment by chartgerink** [comment]
Thanks @joshwlambert for the suggestion! 

We have the early/middle/late perspective on the package - would this perspective be in addition that? More perspectives might make it more confusing, except if there is a clearly documented wish from people using Epiverse for something like this. Which leads me to ask:
Was this a hypothetical user need (predicted by the group) or a documented user need?

**Reply by joshwlambert** [reply]
Good point @chartgerink. I think the epi-centric theming around early-, middle- and late-tasks should remain the main categorisation of Epiverse packages. But I thought that somewhere else on the website there could be a place to find all packages that provide, for example, simulation functionality. This might help users that do not realise that some packages provide simulation functions as well as other functionality, e.g. {epichains} and {serofoi}.

**Reply by chartgerink** [reply]
Thanks @joshwlambert - that helps me understand :blush:

We're currently working on a more general mapping effort for the tools in epidemiology, so this may be addressed by that. I will check back in once we have this 👍 Feel free to ping if it is taking long 😅
